mayastor/src/bdev/dev/loopback.rs

```diff
@@ -78,6 +78,18 @@ impl CreateDestroy for Loopback {
     }
 
     async fn destroy(self: Box<Self>) -> Result<(), Self::Error> {
+        instances().iter_mut().for_each(|n| {
```

Comment:
I see code like this at various places, it might be worth it to extract this into a method?

Reply:
I have extracted this into the _lookup_child_from_bdev()_ function